### ISSUE TITLE
Feature request: CAME optimizer

### ISSUE THREAD

**Issue by Ynead** [gazingstars123/Anima-Standalone-Trainer#41]
Hi, 

Would it be possible for you to implement CAME optimizer please?

**Comment by Ynead** [comment]
I got it, it was as simple as changing the index.html file to add CAME + pip install came-pytorch !

**Comment by And233** [comment]
have you got some result? i add the CAME too, but it will show nan in loss.
ok, it's the problem of flash-attention, we shall not enable it

**Comment by Ynead** [comment]
Yeah I got some pretty good result. I needed to use a lower LR than when using or other optimizer though. CAME works great for styles and learns much more than other optimizer without really overfitting. 

I haven't encountered any issue with flash attention though.

**Comment by mariohyder1992** [comment]
I can confirm FA gives nans when paired with CAME, tried a quick fix with Claude but it failed. Training is 30% slower without it for me so I had to swap for https://github.com/67372a/LoRA_Easy_Training_Scripts

**Comment by And233** [comment]
> I can confirm FA gives nans when paired with CAME, tried a quick fix with Claude but it failed. Training is 30% slower without it for me so I had to swap for https://github.com/67372a/LoRA_Easy_Training_Scripts

you shall downgrade your FA version and try again. I can work well with 2.7.4 instead of 2.8.3
